app/views/groups/index.html.erb

```diff
@@ -9,10 +23,23 @@
 
   <tbody>
     <% @groups.each do |group| %>
-      <tr>
-        <td><%= group.id %></td>
+      <tr class="<%= group.neighborhood.id %> <%= group.kind %>">
```

Comment:
This line is giving me an error locally (after re-installing the bundle and migrating db):

<img width="1440" alt="screen shot 2015-11-07 at 4 16 06 pm" src="https://cloud.githubusercontent.com/assets/3770344/11018045/df6c7198-856a-11e5-9e1e-4acacca7ca9e.png">

Did you ever see this error @HacksForHugs?